x-pack/plugins/monitoring/server/types.ts

```diff
@@ -144,7 +160,8 @@ export interface LegacyServer {
   };
 }
 
-export type Cluster = ElasticsearchModifiedSource & {
+export type Cluster = Omit<ElasticsearchModifiedSource, 'timestamp'> & {
```

Comment:
I did also try to make `timestamp` optional on the `ElasticsearchModifiedSource` type but that had more ramifications than just making it optional on the more specific `Cluster` type (still had some ramifications, but less).